Dockerfile.osmosis

```diff
@@ -1,27 +1,26 @@
-FROM golang:1.17-alpine3.15 AS osmosis-builder
+FROM golang:1.18-alpine3.15 AS builder
 
 WORKDIR /src/app/
 
-RUN apk add --update make curl git bash gcc linux-headers ca-certificates build-base libc-dev eudev-dev 
-
-# CosmWasm: see https://github.com/CosmWasm/wasmvm/releases
-# 1) Add the releases into /lib/
-ADD https://github.com/CosmWasm/wasmvm/releases/download/v1.0.0/libwasmvm_muslc.aarch64.a \ 
-    https://github.com/CosmWasm/wasmvm/releases/download/v1.0.0/libwasmvm_muslc.x86_64.a \ 
-    /lib/
-# 2) Verify their hashes
-RUN sha256sum /lib/libwasmvm_muslc.aarch64.a | grep 7d2239e9f25e96d0d4daba982ce92367aacf0cbd95d2facb8442268f2b1cc1fc; \
-    sha256sum /lib/libwasmvm_muslc.x86_64.a | grep f6282df732a13dec836cda1f399dd874b1e3163504dbd9607c6af915b2740479
-# Copy the right library according to architecture. The final location will be found by the linker flag `-lwasmvm_muslc`
-RUN cp /lib/libwasmvm_muslc.$(uname -m).a /lib/libwasmvm_muslc.a
+RUN set -eux; apk add --no-cache ca-certificates build-base; apk add git linux-headers
 
 RUN git clone https://github.com/osmosis-labs/osmosis.git \
     && cd osmosis \
-    && git checkout 4800c2be7 \
-    && BUILD_TAGS=muslc LINK_STATICALLY=true make build
+    && git checkout 4800c2be7 
```

Comment:
These are the latest hashes I have (from ~5 days ago) for our dependencies

Should these match those used in docker? Possible we upgraded to a new Osmosis hash since then

```
ICQ: 2ddedc5
hermes: 4e83aae8
juno: 9558920
osmo: 4176b287
gaia: bfb8f8ce
stride: 35c8d77e5286389f373a02d3c0515decaa1587bd
```

Reply:
I think we did upgrade osmosis last minute to 08669da8509059980dc964976ee1ca60c84f5c8a. Not sure on the other ones. We should definitely get on the same page